perf/studio/sanity.config.ts

```diff
@@ -0,0 +1,16 @@
+import {defineConfig} from 'sanity'
+import {deskTool} from 'sanity/desk'
+import {STUDIO_DATASET, STUDIO_PROJECT_ID} from '../config/constants'
+import {simple} from './schema/simple'
+import {deepObject} from './schema/deepObject'
+import {deepArray} from './schema/deepArray'
+
+type FIXME = any
+export default defineConfig({
+  plugins: [deskTool() as FIXME /* 🤷 */],
```

Comment:
Wish I knew! I'll double check if it's still an issue and add a better comment if so ☺️

Reply:
Added a better workaround and a comment here: https://github.com/sanity-io/sanity/blob/48f73c03fc9d748afc784afde79f3d72bdd5ac3c/perf/studio/sanity.config.ts#L10-L12

Might be worth looking into why this happens though (or it might be just a artifact of this particular setup)